unity-client/Assets/UnityGLTF/Scripts/Loader/WebRequestLoader.cs

```diff
@@ -66,53 +60,31 @@ private IEnumerator CreateHTTPRequest(string rootUri, string httpRequestPath)
             {
                 finalUrl = Path.Combine(rootUri, httpRequestPath);
             }
-            
-            UnityWebRequest www = null; 
-            bool retry = true;
-            
-            while (retry)
-            {
-                retry = false;
-                www = new UnityWebRequest(finalUrl, "GET", new DownloadHandlerBuffer(), null);
 
-                www.timeout = 5000;
-#if UNITY_2017_2_OR_NEWER
-                yield return www.SendWebRequest();
-#else
-            yield return www.Send();
-#endif
-                if ((int)www.responseCode >= 400)
+            return DCL.WebRequestController.i.Get(
```

Comment:
`UnityGLTF` should stand by itself because we will have to migrate it to an external repo in some point, so please let's use dependency injection to inject this implementation instead of using `WebRequestController.i`.

Reply:
To do this, we should pass the interface implementation through `GLTFSceneImporter`. This is injected via `GLTFComponent` or `GLTFImporter` (for editor mode). 

I think the editor code path doesn't use the `WebRequestLoader` so we may escape the injection of that path, but we'd have to make sure.

Reply:
@BrianAmadori I have done the changes but I would like to review it with you before commit. Can we get a quick talk?

Reply:
Done in https://github.com/decentraland/explorer/pull/2256/commits/8d8a0aa9218abdb7456ad945675d5c1a30215979